core/arch/arm/include/arm.h

```diff
@@ -75,6 +75,9 @@
 #define ARM32_CPSR_IT_MASK1	0x06000000
 #define ARM32_CPSR_IT_MASK2	0x0000fc00
 
+/* ARM Generic timer definitions */
+#define TIMER_PCT_USR_EN	(1 << 0) /* physical counter el0 access enable */
+#define TIMER_VCT_USR_EN	(1 << 1) /* virtual counter el0 access enable */
```

Comment:
sorry I missed that: `BIT(1)` and `BIT(1)` are better.
Also, Maybe better prefix these `ARM_CNT_`, or `ARM_GENTIMER_`.

Reply:
The names should be `CNTKCTL_PL0PCTEN` and `CNTKCTL_PL0VCTEN` to be consistent with the arm v7 reference manual, unfortunately the names in the v8 reference manual are `CNTKCTL_EL0PCTEN` and `CNTKCTL_EL0VCTEN`. I think we should choose the v7 naming and stick to that.

Reply:
@jenswi-linaro @etienne-lms fixed, thanks

Reply:
@etienne-lms I will also update commit message with proper definitions (example of usage)